CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Versions
 
-## x.x.x
+## 6.3.2
```

Comment:
Now that I think of it, we should probably bump the minor version number to match the underlying native SDKs. We started doing that for our Unity Plugin recently. Since this bump is 12.3.1 -> 12.4.0, the plugin version would change to 6.4.0. I think that is the convention we found.